src/plugins/data/server/ui_settings.ts

```diff
@@ -324,8 +324,8 @@ export function getUiSettings(
         defaultMessage: 'Time filter refresh interval',
       }),
       value: `{
-  "pause": false,
-  "value": 0
+  "pause": true,
+  "value": 10000
```

Comment:
@elastic/kibana-app-services Can you please approve this? It looks like a mistake that until now we've had auto-refresh enabled by default with a value of 0.

This was brought up [years ago](https://github.com/elastic/kibana/issues/70562#issuecomment-1070919143) but it looks like it slipped through the cracks. At this time I assume most solutions are manually overriding Kibana's behaviour.